src/main/java/mate.academy/service/ProductParser.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
The `@Component` annotation is applied to an interface. Typically, `@Component` should be used on concrete implementation classes rather than interfaces. Consider moving this annotation to the implementing class.